### PR TITLE
(Assist) Updating to match Core CMS settings structure

### PR DESCRIPTION
**Pull request by tacc-wbomar**
## To Do

<details>
<summary>Completed (Includes Server Testing)</summary>

- [x] Confirm setting names are up-to-date.
- [x] Confirm setting values are up-to-date.
	- [x] A2CPS — J.G., W.B.
	- [x] Core — J.G.
	- [x] Example — W.B.
	- [x] Frontera — J.G., W.B.
	- [x] LCCF — W.B.
	- [x] Neuronex — W.B.
	- [x] ProTX — W.B., J.G.
	- [x] Scivis Color — W.B.
	- [x] TAPIS — W.B.
	- [x] Texascale — W.B.
	- [x] TUP — J.G.
- [x] Test now.
	- [x] A2CPS — J.G.
	- [x] Core — J.G.
	- [x] ProTX — J.G.
	- [x] TUP — J.G.

</details>

<details open>
<summary>Testing For Later</summary>

- [x] UTRC — J.G., W.B. (via #80)
- Texascale — W.B.
- 3dem
- et cetera

</details>

## Overview

Update `task/FP-1194` with what I perceived to be missing changes.

## Changes

- Make changes missing for A2CPS and TUP CMS.
- Rename `PORTAL` to `INCLUDES_CORE_PORTAL` (like in https://github.com/TACC/Core-CMS/pull/341).

## Screenshots (Abridged)

<details><summary>Incomplete Set</summary>

Screen Shot 2021-09-17 at 4 54 11 AM
Screen Shot 2021-09-17 at 8 24 39 PM

</details>

## Testing (Abridged)

1. Load server (local/remote) with given secrets.
     - **Remote**: Rely on build/deploy infrastructure.
     - **Local**: Copy `taccsite_custom/PROJECT/settings_custom.py` to `taccsite_cms//settings_custom.py`.

## Notes

~~@xandroc If the state of `task/FP-1194`, without these changes, is intentional, then I'll just close this out.~~ [These changes have proven to be necessary, through John's testing.]

### CODE REVIEW

File: protx-cms/settings_custom.py [hunk 1]
```diff
@@ -77,10 +87,8 @@
...
 ]
 
-LOGO = PORTAL_LOGO
-
 ########################
 # PORTAL
 ########################
 
-PORTAL = True
+INCLUDES_CORE_PORTAL = True
```
Comment (tacc-wbomar) [inline protx-cms/settings_custom.py:94]:
```suggestion
INCLUDES_CORE_PORTAL = True

########################
# EXPORT
########################

SETTINGS_EXPORT = [
    'BRANDING',
    'LOGO',
    'FAVICON',
    'GOOGLE_ANALYTICS_PROPERTY_ID',
    'GOOGLE_ANALYTICS_PRELOAD'
]
```

File: tup-cms/settings_custom.py [hunk 2]
```diff
@@ -1,57 +1,31 @@
...
+    ('guides/data_transfer.html', 'Guide: Data Transfer'),
+    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
+    ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
 )
 
-########################
-# GOOGLE ANALYTICS
-########################
-
-_GOOGLE_ANALYTICS_PROPERTY_ID = "UA-96034853-2"
```
Comment (tacc-wbomar) [inline tup-cms/settings_custom.py:27]: @taoteg I noticed [you had asked about this property](https://github.com/TACC/Core-CMS-Resources/pull/74/commits/4044081bd516440efa5b7246bfd51ec53dd142ac#diff-09fb7241546cad246d7fbb69434ba551418ca43d62e9cd6eb1f3f25e9d860345R24).

It is the current ID used on https://www.tacc.utexas.edu (found by searching markup for `UA-`).
- I put it in assuming we'd not create a new property in Google Analytics.
- I do not have access to this property.

File: tup-cms/settings_custom.py [hunk 3]
```diff
@@ -1,57 +1,31 @@
...
-_CMS_TEMPLATES = (
-    ('tup-cms/templates/fullwidth.html', 'Fullwidth')
+CMS_TEMPLATES = (
+    ('tup-cms/templates/fullwidth.html', 'Fullwidth'),
+    ('guide.html', 'Guide'),
+    ('guides/getting_started.html', 'Guide: Getting Started'),
+    ('guides/data_transfer.html', 'Guide: Data Transfer'),
+    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
+    ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
 )
```
Comment (tacc-wbomar) [inline tup-cms/settings_custom.py:16]: The guides templates are probably not needed.
- guides were always a short term fix
- only intended for portal
- should not be unnecessary when we have a main TACC docs

I suggest either:

- [your initial plan to have them, but commented](https://github.com/TACC/Core-CMS-Resources/pull/74/commits/12c6d24)

	```suggestion
	CMS_TEMPLATES = (
 		('fullwidth.html', 'Full Width'),
 		('tup-cms/templates/fullwidth.html', 'Full Width'),
 
	    # ('guide.html', 'Guide'),
	    # ('guides/getting_started.html', 'Guide: Getting Started'),
	    # ('guides/data_transfer.html', 'Guide: Data Transfer'),
	    # ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
	    # ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
	)
	```

- or to remove them

	```suggestion
	CMS_TEMPLATES = (
 		('fullwidth.html', 'Full Width'),
 		('tup-cms/templates/fullwidth.html', 'Full Width'),
	)
	```

<details>
<summary>Note to Self</summary>

In https://github.com/TACC/Core-CMS-Resources/pull/78/files, the `tup-cms` template will be removed. Doing so here may cause confusion or unnecessary delay.

</details>

**Review by tacc-wbomar — COMMENTED** [review]
I have some comments because they'd be good reference if asked. But I will resolve them immediately, so we can move on. (There's more work after this anyway that will address them.)

**Review by taoteg — APPROVED** [review]
Working as advertised in all deployed tests. =)

**Pull request merged**